server/app.js

```diff
@@ -50,7 +50,7 @@ if (process.env.NODE_ENV === 'development') {
   app.use(connectLiveReload())
 }
 /** favicon setup */
-app.use(favicon(path.join(__dirname, '../public/favicon.ico')))
+app.use(favicon(path.join(__dirname, '../public/img/favicon.ico')))
```

Comment:
Why was this change necessary?

Reply:
We should have a directory structure that is specific to where things are. It makes sense for images to go in an img directory. This was how it was before as well.